edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Core/IBackupSource.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Core
+{
+    using System.Threading.Tasks;
+    public interface IBackupSource
```

Comment:
I would name this IDeploymentBackupSource since its so specific

Reply:
Sounds good, Done.